server/src/main/java/org/elasticsearch/search/aggregations/support/AggregationContext.java

```diff
@@ -187,6 +189,11 @@ public MappedFieldType getFieldType(String path) {
             return context.getFieldType(path);
         }
 
+        @Override
+        public boolean isFieldMapped(String field) {
```

Comment:
you may also remove getMapper from this class , one less usage of getMapperService ;)